lms/djangoapps/courseware/views/views.py

```diff
@@ -234,6 +222,23 @@ def _downloadable_cert_data(download_url=None, cert_web_view_url=None):
     )
 
 
+def _unverified_cert_data(cert_downloadable_status):
```

Comment:
This function doesn't need cert_downloadable_status as an argument, because it doesn't use it. A function with no arguments is sufficient.

Reply:
Corrected